api-spec/protobuf/tdexa/v1/analytics.proto

```diff
@@ -88,6 +88,10 @@ message MarketPrice {
   double quote_reference_price = 4;
   // point in time when market had this price
   string time = 5;
+  // average price for the specified time_frame
+  double average_price = 6;
+  // average price converted to reference one
+  double average_reference_price = 7;
```

Comment:
These fields must be moved to `MarketPricesReply` message

Reply:
based on offline conversation, these fields must still be moved to `MarketPricesReply` and should be defined as:
```
message MarketPricesReply {
   [...]
   repeated AveragePrice average_price = 6;
}

message AveragePrice {
   repeated string market_ids = 1;
   double average_price = 2;
   double average_reference_price = 2;
}
```